### DISCUSSION TITLE
Command /home

### DISCUSSION THREAD

**Discussion by Aides123** [Zrips/CMI#5848] [Q&A]
Hi! For some time now, you have to type / home <home_name>
Even if the player only has one house. How do I make it enough to type /home?
I mean:
The player type /home and is moved to his home without specifying the exact house name.

**Comment by mrfloris** [comment]
Hi there, to my knowledge it already does this.

I've deleted all the homes from a test user, and set one home. Then when I type /home, it goes to that location. 

<img width="341" alt="Screenshot 2021-09-08 at 09 36 08" src="https://user-images.githubusercontent.com/28841349/132466641-5b339a72-62b9-40bc-a0dc-a25bf2f7d9ea.png">

<img width="591" alt="Screenshot 2021-09-08 at 09 36 21" src="https://user-images.githubusercontent.com/28841349/132466659-a82bbb86-af5c-4be7-b095-b52e08810c08.png">

i walked away from the location. 

<img width="437" alt="Screenshot 2021-09-08 at 09 37 48" src="https://user-images.githubusercontent.com/28841349/132466743-c8dc9209-00ae-44f6-94f8-48734a252b92.png">

And did `/cmi home` and it just teleported me to the 'mybase' location i previously set (my only home).
